src/vmm/src/arch/x86_64/mod.rs

```diff
@@ -35,7 +35,7 @@ use linux_loader::loader::elf::start_info::{
 
 use crate::arch::{BootProtocol, InitrdConfig, SYSTEM_MEM_SIZE, SYSTEM_MEM_START};
 use crate::device_manager::resources::ResourceAllocator;
-use crate::utils::u64_to_usize;
+use crate::utils::{MIB_TO_BYTES_SHIFT, u64_to_usize};
```

Comment:
Maybe we should just add a `mb_to_bytes` function to the `utils` module?

Reply:
mh, yeah, that's probably shorter even, let me change :)

Reply:
done